### DISCUSSION TITLE
Get enhanced stereo fields from CXSmiles

### DISCUSSION THREAD

**Discussion by zackterray** [rdkit/rdkit#6410] [Q&A] [answered]
I was wondering if there was a way to get a string of just the enhanced stereo information (excluding other cxsmiles fields) from a rdkit mol generated from a cxsmiles.  I.e. output CXSmiles with just the enhanced stereo group information.  Alternatively, if there is a way to get rdkit's inchi to include enhanced stereochemistry, that would also work. Thanks!

**Comment by kienerj** [comment, marked as answer]
See class [rdkit.Chem.rdmolfiles.CXSmilesFields](https://www.rdkit.org/docs/source/rdkit.Chem.rdmolfiles.html#rdkit.Chem.rdmolfiles.CXSmilesFields).

```python
write_params = Chem.SmilesWriteParams()
Chem.MolToCXSmiles(mol, params=write_params, flags=Chem.rdmolfiles.CXSmilesFields.CX_ENHANCEDSTEREO)
```

**Reply by zackterray** [reply]
Perfect! Thank you!
